SalesforceCommerceCloud/pwa-kit
# [V2][Multi-site]Fix the case when no site aliases is set

# Description
When no site aliases are set, the wrong site is returned. Inside `resolveSiteFromUrl` logic, siteRef is undefined when we are on the homepage, and when the site alias is also undefined, it happens to return a truthy value, which is wrong. 



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

-In your app config,  remove site aliases in your retail app config. It should be an empty object
- `npm start` in retail app
- Verify that `RefArchGlobal` is correct site being used when you are in home page. You should also see it in the urls as such `RefArchGlobal/en-US/womens/dresses

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
